api/prisma/schema.prisma

```diff
@@ -46,6 +46,7 @@ model WorksheetCourses {
   crn             Int    @db.UnsignedMediumInt
   season          Int    @db.UnsignedMediumInt
   worksheetNumber Int    @default(0) @db.UnsignedMediumInt
+  color           String @db.VarChar(32)
```

Comment:
using string should probably be fine, but there is some native hex support (the benefits of which i haven't totally looked into)
https://stackoverflow.com/questions/4306219/inserting-hex-value-in-mysql

Reply:
(if we ever allow users total control over the color, this could be helpful to make sure we aren't storing non hex values in this column)

Reply:
Nah, we aren't _just_ going to use hex. If user specified a `hsl` color we are going to use that.

Reply:
It's not going to be lossless so I'd rather not. I'm confident that precisely zero person is going to care but just to be safe.

Reply:
kk sounds good.  we should do some backend validation too though that what is storing is valid

Reply:
Good idea